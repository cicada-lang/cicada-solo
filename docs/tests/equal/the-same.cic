the_same(String, "a")
