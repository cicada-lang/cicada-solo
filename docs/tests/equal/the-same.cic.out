refl(vague String, vague "a"): Equal(String, "a", "a")